Automattic/jetpack
# WAF: Introduce logging when a request is blocked

#### Changes proposed in this Pull Request:

* Logs to file and to database when a request is blocked.

#### Jetpack product discussion


1201069996155224-as-1201965730420999

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Checkout this branch
* Build the plugin with it's dependencies
* Activate it on your test environment
* Make sure that the WAF log directory was created
* Test a blocked request and verify that it was logged in the file and in the database.
